### PR TITLE
building component class and styling in scss, 9&10 UI component button

### PR DESCRIPTION
**Pull request by szarlata**
New component - red button.
JS: component created as a class RedButton. Inner text can be passed as an argument of the constructor. renderElement included.
SCSS: styling for the component.

### CODE REVIEW

**Comment by szarlata** [comment]
I'm not sure what method render() should return.

**Comment by DaveTheWebDev** [comment]
It should return button you have created, do you understand my code proposition?

**Comment by szarlata** [comment]
I added some comments in js file to show my thinking process. I would be super glad if you could check my approach.

File: src/app/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    };
+    
+    render() {
+      const button = renderElement({
+        tagHTML: "button",
+        className: `${this.className}`,
+        innerText: `${this.text}`
+     });
+  
+      return button;
```
[the remark this answers is not in the document]
Comment (szarlata) [reply src/app/components/Button.js:16]: I thought "return button" is in render() method? I close the method with "}" after "return button" and then I close the class declaration.

[the remark this answers is not in the document]
Comment (ZibiH) [reply src/app/components/Button.js:16]: True, my bad :)

**Review by ZibiH — COMMENTED** [review]
I like how it looks right now!

**Pull request merged**